<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QMainWindow" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="mainLayout" stretch="3,1">
      <item>
       <layout class="QHBoxLayout" name="topLayout" stretch="1,3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="SourceTab">
           <attribute name="title">
            <string>数据源</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>数据来源</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="DataSourceList"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>COM口</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="COMList"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>波特率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="BaudrateList"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,3">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>描述</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="SourceDetail"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="ConnectButton">
                <property name="text">
                 <string>连接</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>267</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="ObserverTab">
           <attribute name="title">
            <string>观察</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QListView" name="ObservedList"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QToolButton" name="AddButton">
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="DeleteButton">
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="Chart"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="RawData"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
